cli/build.go

```diff
@@ -13,37 +13,31 @@ import (
 	"github.com/uor-framework/client/builder"
 	"github.com/uor-framework/client/builder/graph"
 	"github.com/uor-framework/client/builder/parser"
-	"github.com/uor-framework/client/registryclient/orasclient"
 	"github.com/uor-framework/client/util/workspace"
 )
 
 type BuildOptions struct {
 	*RootOptions
-	Destination string
-	RootDir     string
-	Insecure    bool
-	PlainHTTP   bool
-	Configs     []string
-	Output      string
-	Push        bool
+	RootDir string
+	Output  string
 }
 
 var clientBuildExamples = templates.Examples(
 	`
-	# Template content in a directory without pushing
-	client build <directory>
+	# Template content in a directory
+	client build mydirectory
```

Comment:
In the output of the command, do we want to mention where the workspace directory is located?

Reply:
Done.